core/model/src/main/java/it/unibz/inf/ontop/model/term/functionsymbol/CaseSPARQLFunctionSymbol.java

```diff
@@ -0,0 +1,4 @@
+package it.unibz.inf.ontop.model.term.functionsymbol;
+
+public interface CaseSPARQLFunctionSymbol {
```

Comment:
Could you please make it extend an existing interface?

Reply:
I suggest renaming it into `CasingSPARQLFunctionSymbol`, to avoid the confusion with the SQL `CASE` expression.